MODULE.bazel

```diff
@@ -30,7 +30,6 @@ rules_kotlin_bootstrap_extensions = use_extension(
 )
```

Comment:
should rules_kotlin also be updated from 1.8.1 to 1.9.6?

Reply:
For completeness, yes.

Practically, it doesn't do anything.